server/utils/assessmentUtils.ts

```diff
@@ -25,35 +25,49 @@ export const formatAssessmentData = (
     .map(areaConfig => {
       let score
-      const isLinkedtoRoSH = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToHarm`] === 'YES'
-      const isLinkedtoReoffending =
-        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToReoffending`] === 'YES'
-      const isLinkedtoStrengthsOrProtectiveFactors =
-        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}Strengths`] === 'YES'
+
+      const linkedtoHarm = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToHarm`].toLowerCase()
+      const linkedtoReoffending =
+        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}LinkedToReoffending`].toLowerCase()
+      const linkedtoStrengthsOrProtectiveFactors =
+        crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}Strengths`].toLowerCase()
+
       let subData: SubAreaData
       let overallScore
       const motivationToMakeChanges = motivationText(
         assessment.sanAssessmentData[`${areaConfig.assessmentKey}_changes`]?.value,
       )
 
-      const linkedToRoSHYesNo = isLinkedtoRoSH ? 'yes' : 'no'
-      const linkedToReoffendingYesNo = isLinkedtoReoffending ? 'yes' : 'no'
-      const linkedtoStrengthsOrProtectiveFactorsYesNo = isLinkedtoStrengthsOrProtectiveFactors ? 'yes' : 'no'
-
       const riskOfSeriousHarmDetails =
-        assessment.sanAssessmentData[
-          `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_serious_harm_${linkedToRoSHYesNo}_details`
-        ]?.value
+        linkedtoHarm !== 'null'
+          ? assessment.sanAssessmentData[
+              `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_serious_harm_${linkedtoHarm}_details`
+            ]?.value
+          : undefined
+
       const riskOfReoffendingDetails =
-        assessment.sanAssessmentData[
-          `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_reoffending_${linkedToReoffendingYesNo}_details`
-        ]?.value
+        linkedtoReoffending !== 'null'
+          ? assessment.sanAssessmentData[
+              `${areaConfig.assessmentKey}_practitioner_analysis_risk_of_reoffending_${linkedtoReoffending}_details`
+            ]?.value
+          : undefined
+
       const strengthsOrProtectiveFactorsDetails =
-        assessment.sanAssessmentData[
-          `${areaConfig.assessmentKey}_practitioner_analysis_strengths_or_protective_factors_${linkedtoStrengthsOrProtectiveFactorsYesNo}_details`
-        ]?.value
+        linkedtoStrengthsOrProtectiveFactors !== 'null'
+          ? assessment.sanAssessmentData[
+              `${areaConfig.assessmentKey}_practitioner_analysis_strengths_or_protective_factors_${linkedtoStrengthsOrProtectiveFactors}_details`
+            ]?.value
+          : undefined
 
       score = crimNeeds[areaConfig.crimNeedsKey][`${areaConfig.crimNeedsSubKey}OtherWeightedScore`]
+
+      const isMissingInformation =
```

Comment:
if we can make these all either null or undefined (by making the code above which sets the values consistent) then you can do something like `isMissingInformation = [linkedToHarm, LinkedToReoffend, etc].every(field => field !== null)`